### ISSUE TITLE
The Total Power Usage value is displayed as 0 after a new login

### ISSUE THREAD

**Issue by roadie33** [RealTryZ/Wattometer#4]
I access my Octopi with several browsers on different devices. Especially when I am not at home during a print. I noticed that the value of the total power consumption is displayed as 0 when I log in again. Shouldn't the value be displayed regardless of the login during a print? _(Also, I would like the value to be displayed until the next print starts or another print is selected. -> Fixed by issue #2)_
![grafik](https://github.com/RealTryZ/Wattometer/assets/58664090/e2230e9d-e2c9-4c51-9f95-84a03d0233e2)

**Comment by roadie33** [comment]
Posting edited, because my last Question is fixed by issue #2

**Comment by RealTryZ** [comment]
Hey,👋
first of thank you for the feedback.
I will look in the following days how I can achieve that the total power usage will be saved through multiple sessions😊

**Comment by RealTryZ** [comment]
Hey,
I redesigned the plugin so it now saves the total power usage and displays it in multiple browser or after browser refresh.
Would love to get some feedback :)

**Comment by roadie33** [comment]
Hello,
First of all, thank you very much for quickly providing a version in which the issue has been fixed. After receiving the new version, I installed and tested it immediately. 
For the test I started a print to access the server from another browser. The total power consumption was also displayed in this browser. Both browsers displayed the identical value. I then accessed the server via VPN with a third browser. Here too, the value was displayed identically to the other two browsers. As a final test, I deleted the history, all cookies and saved passwords in one browser. When I logged in again, the value was displayed again. From my point of view, the issue is fixed.
